ibis/expr/api.py

```diff
@@ -171,6 +171,13 @@
 dtype = dt.dtype
 infer_dtype = dt.infer
 infer_schema = sch.infer
+aggregate = ir.Table.aggregate
+cross_join = ir.Table.cross_join
+join = ir.Table.join
+asof_join = ir.Table.asof_join
```

Comment:
I'm a little dubious on these, but they're also maybe the way some people like to write joins.

Reply:
Yeah, I could go either way here. Since they're table-level operations I felt fine leaving them for now.